examples/model_selection/plot_grid_search_refit_callable.py

```diff
@@ -0,0 +1,113 @@
+"""
+==================================================
+Balance model complexity and cross-validated score
+==================================================
+
+This example balances model complexity and cross-validated score by
+finding a decent accuracy within 1 standard deviation of the best accuracy
+score while minimising the number of PCA components.
```

Comment:
This is well documented in the elements of statistical learningl (IIRC). I guess it wouldn't hurt to cite that chapter.